src/components/modal/modal.scss

```diff
@@ -22,7 +22,7 @@ $modal-inset-height-large:                   600px !default;
 
   ion-backdrop {
     @media not all and (min-width: $modal-inset-min-width) and (min-height: $modal-inset-min-height-small) {
-      display: none;
+      visibility: hidden;
```

Comment:
Note, we'll need to change this back to display:none when the Animation class checks if an element is displayed or not.